architecture/distribution-layer.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Distribution Layer
+summary: 
+toc: false
+---
+
+Present a unified view of many replicated KV ranges.
+
+<div id="toc"></div>
+
+## Overview
+
+CockroachDB is a distributed, fault-tolerant database, which means that you must have data spread among many nodes in the same cluster; all of which is replicated at least 3 times. However, all of the data still needs to be easily accessible. To achieve this Cockroach uses a monolithic sorted map, which is an order-preserving distribution.
```

Comment:
Map makes sense, my question was around the word "distribution"

Reply:
Maybe this is a term of art to oppose it to a [distributed hash table](https://en.wikipedia.org/wiki/Distributed_hash_table)? I hesitate to change it because the architecture doc uses this term heavily.

@bdarnell Do you have any insight/recommendation?

Reply:
I agree with @nstewart, "data structure" is better than "distribution" here. (and elsewhere - this is the "distribution layer", but the sorted map is not "a distribution")

Reply:
Done.